app/models/manageiq/providers/base_manager.rb

```diff
@@ -64,9 +64,5 @@ def self.default_blacklisted_event_names
     def self.options_description
       {}
     end
-  end
-
-  def self.display_name(number = 1)
-    n_('Manager', 'Managers', number)
-  end
-end
+  end # BaseManager class
+end   # ManageIQ::Providers module
```

Comment:
Can you remove these comments?  They are going to get removed with the rubocop changes anyway.

Reply:
I put them because the file has an abnormal layout and bit us in cfdd29b405b86faa2e9bcc3d4d3a0ea72082f8e1. Ideally, we fix the layout.

Reply:
yeah that layout is pretty standard

Reply:
I appeased the annoying cop.  Hopefully, no one gets bit by the scoping.